wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/ProjectManager.java

```diff
@@ -320,6 +341,23 @@ public RegisteredProject updateProject(ProjectConfig newConfig) throws Forbidden
         return project;
     }
 
+    /**
+     *
+     * Import source code as a Basic type of Project
+     *
+     * @param path where to import
+     * @param sourceStorage where sources live
+     * @param rewrite whether rewrite or not (throw exception othervise) if such a project exists
+     *
+     * @return Project
+     *
+     * @throws ServerException
```

Comment:
exceptions should be written: what is the cause of all of them
these format is useless